Interfaces/Website/ArabicTextDetection/index.html

```diff
@@ -0,0 +1,72 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <title>القارئ</title>
+
+    <link
+      rel="stylesheet"
+      href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css"
+      integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T"
+      crossorigin="anonymous"
+    />
+    <link rel="stylesheet" href="index.css" />
+    <script src='https://unpkg.com/tesseract.js@4.0.1/dist/tesseract.min.js'></script>
+  </head>
+  <body>
+    <div class="container" id="maincontainer">
+      <div class="av-container">
+        <div id="avatar"> القَارِئ</div>
+      </div>
+      <div class="info text-right">
+        <h3>:الفكرة</h3>
+        <p>
+          يقوم هذا النموذج بالتعرف على النص الموجود في الصور 
+        </p>
+      </div>
+      <div id="loader">
+        <div class="lds-grid">
+          <div></div>
+          <div></div>
+          <div></div>
+          <div></div>
+          <div></div>
+          <div></div>
+          <div></div>
+          <div></div>
+          <div></div>
+        </div>
+        الرجاء الانتظار لتهيئة النموذج
+      </div>
+      <div class="inputs text-center collapse" id = "inputs">
+          <canvas id = "canvas" width = "512px", height = "512px"></canvas>
+          <br>
+          <input type="file" class = "btn btn-dark btn-sm preview collapse" id = "preview">
+
+          <button class="btn btn-dark btn-sm submit">! تعرف </button>
```

Comment:
It seems that you uploaded other images, can we switch between them to allow more testing?

Reply:
Yes. Now the image is selected randomly from 6 available images. 
Also added a button to randomly switch the image.